Kudu.Console/packages.config

```diff
@@ -1,12 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="net45" />
-  <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="net45" />
-  <package id="Microsoft.CodeAnalysis.Analyzers" version="2.9.8" targetFramework="net46" developmentDependency="true" />
-  <package id="Microsoft.Diagnostics.Tracing.EventRegister" version="1.0.26" targetFramework="net45" />
-  <package id="Microsoft.Diagnostics.Tracing.EventSource" version="1.0.26" targetFramework="net45" />
-  <package id="Microsoft.Diagnostics.Tracing.EventSource.Redist" version="1.0.26" targetFramework="net45" />
-  <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="net45" />
-  <package id="System.IO.Abstractions" version="1.4.0.74" targetFramework="net45" />
-  <package id="XmlSettings" version="0.1.3" targetFramework="net45" />
-</packages>
```

Comment:
why is this file removed ?

Reply:
Upgraded the project to newer project style that no longer requires the packages.config file.